main.css

```diff
@@ -0,0 +1,97 @@
+@import url(/components/avatar/avatar.css);
+@import url(/components/badge/badge.css);
+
+/* fonts */
+@import url("https://fonts.googleapis.com/css2?family=Montserrat:ital,wght@0,100;0,200;0,300;0,400;0,500;0,600;0,700;0,800;0,900;1,100;1,200;1,300;1,400;1,500;1,600;1,700;1,800;1,900&display=swap%27");
+@import url("https://fonts.googleapis.com/css2?family=Cinzel:wght@400;500;600;700;800;900&display=swap%22");
+/* fonts */
+
+:root {
+  --white-color: #ffffff;
+  --black-color: #000000;
+  --primary-color: #3d348b;
+  --secondary-color: #b8b8ff;
+  --tertiary-color: #f7b801;
+  --online-color: #01af0f;
+  --offline-color: #808080;
+  --busy-color: #ff0000;
+  --notif-color: #f7b801;
+  --primary-font-family: "Cinzel", serif;
+  --secondary-font-family: "Montserrat", sans-serif;
+}
+html {
+  font-size: 62.5%;
+}
+a {
+  color: inherit;
+  text-decoration: inherit;
+}
+body {
+  margin: 0;
+}
+h2,
+p {
+  padding: 0;
+  margin: 0;
+}
+.page__container {
+  display: flex;
+  flex-direction: column;
+  justify-items: center;
+}
+.header {
+  display: flex;
+  flex-wrap: wrap;
+  justify-content: space-between;
+  align-items: center;
+  padding: 0.5rem;
+  font-family: var(--primary-font-family);
+  font-size: 1.2rem;
+  color: var(--black-color);
+  background-color: var(--white-color);
+  position: sticky;
+  top: 0;
+  z-index: 2;
+}
+.header__logo-container {
+  display: flex;
+  align-items: center;
+  gap: 0.5rem;
+  cursor: pointer;
+  margin-left: 1.5rem;
+}
+.header__logo {
+  font-size: 2.2rem;
+}
+.logo__img {
+  width: 40px;
+}
+.header__nav {
+  display: flex;
```

Comment:
You can also make one class for flex as you are using it again and again .. so it will reduce your work also.